API/Configurations/RateLimitConfiguration.cs

```diff
@@ -0,0 +1,19 @@
+using System.ComponentModel.DataAnnotations;
+
+namespace API.Configurations;
+
+public class RateLimitConfiguration
+{
+    public const string Position = "RateLimit";
+
+    /// <summary>
+    /// The total amount of tokens available per <see cref="Window"/>
+    /// </summary>
+    [Range(1, int.MaxValue, ErrorMessage = "PermitLimit must be an integer greater than 1!")]
+    public int PermitLimit { get; } = 30;
+    /// <summary>
+    /// The amount of time (in seconds) before the <see cref="PermitLimit"/> is refreshed
+    /// </summary>
+    [Range(1, int.MaxValue, ErrorMessage = "Window must be an integer greater than 1!")]
+    public int Window { get; } = 30;
```

Comment:
This should be 60 for 30req/min right?

Reply:
Yep, this got changed when I went from a bucket to fixed window limit, and this was the one spot this didn't get changed. If you look at the diff of the first two commits this was literally just renamed from `TokensPerPeriod` to `Window`. Silly mistake